st2tests/tests/configs/tests.yaml.yaml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Shouldn't this file be `st2tests/tests/configs/tests.yaml`?  There's two yamls in the file name.